.github/workflows/build_hub.yml

```diff
@@ -4,6 +4,11 @@ on:
   pull_request:
     branches:
       - master
+    paths: 
```

Comment:
unnecessary: this is short-circuted based on changes detected via build vars
or should we move away from the using the build vars logic

Reply:
We should probably move away from doing path detection if it's simple enough to be covered using the `paths` property in a github action doc.

Reply:
limiting this to run only when there are changes to `prime-router` means this workflow will not run.  `required` workflows must run (even if there is nothing to do).